README.md

```diff
@@ -1,12 +1,5 @@
-# Pipeline Shared Libraries
+# Pipeline Deprecated Groovy Libraries
 
-When you have multiple Pipeline jobs, you often want to share some parts of the Pipeline scripts between them to keep Pipeline scripts [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself).
-A very common use case is that you have many projects that are built in the similar way.
-
-This plugin adds that functionality by allowing you to create “shared library script” SCM repositories.
-It can be used in two modes:
-
--   A legacy mode in which there is a single Git repository hosted by Jenkins itself, to which you may push changes
--   A more general mode in which you may define libraries hosted by any SCM in a location of your choice.
-
-Comprehensive user documentation can be found [in the Pipeline chapter of the User Handbook](https://jenkins.io/doc/book/pipeline/shared-libraries/).
+This plugin is deprecated.
+It implements a legacy mode of defining Groovy libraries in which there is a single Git repository hosted by Jenkins itself, to which you may push changes.
+Instead use **Pipeline: Groovy Libraries**.
```

Comment:
Maybe turn this into a link? Also the display name is "Pipeline: Shared Groovy Libraries":

```suggestion
Instead use [Pipeline: Shared Groovy Libraries](https://plugins.jenkins.io/pipeline-groovy-lib/).
```

Reply:
I meant to get rid of the _shared_ part (which is redundant for a _library_: if it were not shared, why would you need a library?). I will do the link.

Reply:
Yes I agree that "Shared" is redundant, but from a consistency standpoint, existing documentation uses the term "Shared Libraries", e.g. https://www.jenkins.io/doc/book/pipeline/shared-libraries/, and https://github.com/jglick/pipeline-groovy-lib-plugin/pull/5 does not change the display name.